ledger/ledgercore/statedelta.go

```diff
@@ -695,10 +692,10 @@ func (ad AccountDeltas) ApplyToBasicsAccountData(addr basics.Address, prev basic
 
 // GetAllAppResources returns all AppResourceRecords
 func (ad *AccountDeltas) GetAllAppResources() []AppResourceRecord {
-	return ad.appResources
+	return ad.AppResources
 }
 
 // GetAllAssetResources returns all AssetResourceRecords
 func (ad *AccountDeltas) GetAllAssetResources() []AssetResourceRecord {
-	return ad.assetResources
+	return ad.AssetResources
```

Comment:
Alternatively we can just add
```go
func (ad *AccountDeltas) GetAllAccounts() []NewBalanceRecord {
    return ad.accts
}
```
so that we have accessors for all the private fields.